schemas/com.snowplowanalytics.badrows/schema_violations/jsonschema/1-0-0

```diff
@@ -0,0 +1,167 @@
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "failure": {
+      "type": "object",
+      "description": "Information regarding the schema violations",
+      "properties": {
+        "timestamp": {
+          "type": "string",
+          "description": "Timestamp at which the failure occurred",
+          "format": "date-time"
+        },
+        "messages": {
+          "type": "array",
+          "description": "List of failure messages associated with the schema violations",
+          "items": {
+            "type": "object",
+            "description": "Individual schema violation",
+            "anyOf": [
+              {
+                "properties": {
+                  "schemaKey": {
+                    "type": "string",
+                    "description": "Schema which caused the failure",
+                    "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$"
+                  },
+                  "instance": {
+                    "type": "string",
+                    "description": "Where the offending data was found",
+                    "maxLength": 128
+                  },
+                  "igluRepositories": {
+                    "type": "array",
+                    "description": "Iglu repositories tried or where the offending schema was found",
+                    "items": {
+                      "type": "string",
+                      "description": "Name of the Iglu repository",
+                      "maxLength": 128
+                    }
+                  },
+                  "message": {
+                    "type": "string",
+                    "description": "Failure message associated with this schema violation",
+                    "maxLength": 256
+                  }
+                },
+                "required": [ "schemaKey", "instance", "igluRepositories", "message" ],
+                "additionalProperties": false
+              },
+              {
+                "properties": {
+                  "schemaKey": {
+                    "type": "string",
+                    "description": "Schema which was violated",
+                    "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$"
+                  },
+                  "instance": {
+                    "type": "string",
+                    "description": "Where the offending data was found",
+                    "maxLength": 128
+                  },
+                  "igluRepositories": {
+                    "type": "array",
+                    "description": "Iglu repositories tried or where the offending schema was found",
+                    "items": {
+                      "type": "string",
+                      "description": "Name of the Iglu repository",
+                      "maxLength": 128
+                    }
+                  },
+                  "property": {
```

Comment:
Is this equivalent to pointer?

Reply:
no this would be the violated property, e.g. `maxLength` but this schema is really dependent on what is going to end up being in the new iglu client